[2022-05-22 22:13:36,613] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: book_db.save_books 2022-05-22T22:13:10.207470+00:00 [queued]>
[2022-05-22 22:13:36,648] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: book_db.save_books 2022-05-22T22:13:10.207470+00:00 [queued]>
[2022-05-22 22:13:36,649] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 22:13:36,650] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-05-22 22:13:36,651] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 22:13:36,950] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): save_books> on 2022-05-22T22:13:10.207470+00:00
[2022-05-22 22:13:36,956] {standard_task_runner.py:52} INFO - Started process 63 to run task
[2022-05-22 22:13:36,963] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'book_db', 'save_books', '2022-05-22T22:13:10.207470+00:00', '--job-id', '12', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/books_db.py', '--cfg-path', '/tmp/tmp0nm6t5y2', '--error-file', '/tmp/tmp5vl5u9w4']
[2022-05-22 22:13:36,967] {standard_task_runner.py:77} INFO - Job 12: Subtask save_books
[2022-05-22 22:13:37,081] {logging_mixin.py:104} INFO - Running <TaskInstance: book_db.save_books 2022-05-22T22:13:10.207470+00:00 [running]> on host 0f09442c242c
[2022-05-22 22:13:37,285] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=book_db
AIRFLOW_CTX_TASK_ID=save_books
AIRFLOW_CTX_EXECUTION_DATE=2022-05-22T22:13:10.207470+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-22T22:13:10.207470+00:00
[2022-05-22 22:13:37,318] {python.py:151} INFO - Done. Returned value was: None
[2022-05-22 22:13:37,333] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=book_db, task_id=save_books, execution_date=20220522T221310, start_date=20220522T221336, end_date=20220522T221337
[2022-05-22 22:13:37,536] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-05-22 22:13:37,593] {local_task_job.py:151} INFO - Task exited with return code 0
